src/main/frontend/components/ConcertVenueForm.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from "react";
+import { Redirect } from "react-router";
+import _ from 'lodash'
+
+import StateDropdown from "./StateDropdown";
+import ErrorList from "./ErrorList"
+
+const ConcertVenueForm = (props) => {
+
+  const[formPayload, setFormPayload] = useState({
+    name:"",
+    address:"",
+    city:"",
+    state:"",
+    zipCode:"",
+    capacity:"",
+    description:"",
+    imgUrl:"",
+    phoneNumber:""
+  })
+
+  const [errors, setErrors]= useState({})
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [venueId, setVenueId]= useState(null)
```

Comment:
Let's clean up some whitespace here: add a space after each key in your `formPayload` object lines 11-19 and 70-78, and before the equals signs ahead of `useState`